lib/turbolinks.rb

```diff
@@ -33,7 +33,7 @@ def referer
         (ActionView::RoutingUrlFor rescue ActionView::Helpers::UrlHelper).module_eval do
           include XHRUrlFor
         end
-      end unless RUBY_VERSION =~ /^1\.8/
+      end
```

Comment:
Rails 4 requires Ruby 1.9. This was added for Rails 3.2 compatibility: https://github.com/rails/turbolinks/commit/42285201dfb3365dfabe87b4b66a8c7569a07b48